specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/billingSavingsPlansList.json

```diff
@@ -0,0 +1,255 @@
+{
+  "parameters": {
+    "billingAccountName": "00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+    "api-version": "2023-04-01",
+    "$filter": "billingProfileId eq '/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/billingProfiles/AAAA-BBBB-CCC-DDD'",
+    "$orderby": "properties/displayName asc",
+    "refreshSummary": "true",
+    "selectedState": "Succeeded",
```

Comment:
why is this not a filter?

Reply:
`selectedState` is a different property we use in the backend for calculating roll up summary value, so it should be handled separately on RP side

Reply:
Should take this offline for further discussion